src/wallet/wallet.h

```diff
@@ -264,6 +266,9 @@ class CWallet final : public WalletStorage, public interfaces::Chain::Notificati
     void AddToSpends(const COutPoint& outpoint, const uint256& wtxid, WalletBatch* batch = nullptr) EXCLUSIVE_LOCKS_REQUIRED(cs_wallet);
     void AddToSpends(const CWalletTx& wtx, WalletBatch* batch = nullptr) EXCLUSIVE_LOCKS_REQUIRED(cs_wallet);
 
+    void UpdateAddressBookUsed(const CWalletTx&) EXCLUSIVE_LOCKS_REQUIRED(cs_wallet);
+    void InitialiseAddressBookUsed() EXCLUSIVE_LOCKS_REQUIRED(cs_wallet);
```

Comment:
https://github.com/bitcoin/bitcoin/commit/2d3b9e24add7a9eb9824b60fc5590d3a1e4b8c6d suggest ordering these two methods here and in the cpp file with initialize first, then update.

Reply:
Done